corehq/apps/app_manager/suite_xml/generator.py

```diff
@@ -119,12 +119,11 @@ def media_resources(self):
         self.app.remove_unused_mappings()
         if self.app.multimedia_map is None:
             self.app.multimedia_map = {}
-        filter_multimedia = self.app.media_language_map and self.build_profile
+        filter_multimedia = self.build_profile
```

Comment:
This change seems like it will result in multimedia being filtered (or not filtered) in cases where it was not previously. I'm not sure if that's a good or bad thing, and my concentration is too fragmented right now to figure it out.

Reply:
My understanding is that `media_language_map` was never expected to be empty. It would get re-created every time a new version is created with a build profile, but it never gets emptied. So the new logic is a little bit different, but I don't think it's significant. @calellowitz can you confirm?

Reply:
This looks good to me. It sounds like the replacement for `media_language_map` should never be empty.